kong/router/fields.lua

```diff
@@ -197,38 +233,61 @@ if is_http then
   end
 
 
-  setmetatable(FIELDS_FUNCS, {
-  __index = function(_, field)
+  lazy_generate_func = function(funcs, field)
```

Comment:
```suggestion
  get_field_accessor = function(funcs, field)
```

This `lazy` part is not very interesting for the caller, we can make the name a bit more concise. `get` implies it can be generated or simply returned.

Reply:
done.